### PR TITLE
Update .isAlpha() express validator.

### PR DESCRIPTION
**Pull request by vphatfla**
First names and last names should only contain alphabet letters. Using isAlphanumeric() is incorrect.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ exports.author_create_post = [
 
     // Validate and sanitize fields.
     body('first_name').trim().isLength({ min: 1 }).escape().withMessage('First name must be specified.')
-        .isAlphanumeric().withMessage('First name has non-alphanumeric characters.'),
+        .isAlpha().withMessage('First name has non-alphanumeric characters.'),
```
Comment (teoli2003) [inline files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: I'm not sure this is good. _My_ first name contains non-alphabetic characters, and I know plenty of people with non-alphabetic characters in their last names.

`isAlphanumeric()` is incorrect, but `isAlpha()` doesn't solve this.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ 😄

We should not have any of these validations

Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: @teoli2003 @Josh-Cena  You make a good point - sadly Elon Musk won't be able to check in baby "X Æ A-12" in the library.  But wouldn't all of you be happier if all those pointless extra characters were removed from you names ? :-). 

I put in `isAlphanumeric()` to show the use of the validator - almost any validator would have done here. I agree it is a "bad lesson" for programmers, but unless you feel strongly about this I'd rather leave it as is.

Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: Note, I would be very happy to add a note that this is not a good thing to do!

Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: This applies to your request : https://www.youtube.com/watch?v=TkR9QWuoyJ8 :-)

**Comment by hamishwillee** [comment]
Thanks very much for highlighting this issue  @vphatfla 

I'll "fix" it by adding a note rather than altering the method used.

File: files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md (hunk 1 above)
Comment (vphatfla) [reply files/en-us/learn/server-side/express_nodejs/forms/create_author_form/index.md:44]: Thanks @hamishwillee, adding a note would be very useful. :-)

**Comment by hamishwillee** [comment]
DOne!

**Pull request closed without merging**